thyrel-api/thyrel-api/DataProvider/ElementDataProvider.cs

```diff
@@ -169,8 +169,19 @@ public async Task<ElementDto> GetCurrentElement(int playerId)
                     DrawingId = e.DrawingId,
                     FinishAt = e.FinishAt,
                     CreatedAt = e.CreatedAt,
-                    SessionId = e.SessionId
-                }).FirstAsync();
+                    SessionId = e.SessionId,
+                })
+                .Take(2)
+                .ToListAsync();
+
+            var children = elementWithParent[0];
+            ElementDto parent = null;
+            if (elementWithParent.Count() > 1)
+            {
+                parent = elementWithParent[1];
+            }
+
+            var result = new ElementStepDto(children, parent);
```

Comment:
tu peux remplacer ta logic par ça
```suggestion
            var result = new ElementStepDto(children, elementWithParent.Count() > 1 ? elementWithParent[1] : null);
```